django_rest_generator/response.py

```diff
@@ -28,18 +29,21 @@ class APIResponse(object):
     file_name: str
     raw: bytes
 
-    def __init__(self, response: requests.Response) -> None:
+    def __init__(self, response: requests.Response, schema: Schema = None) -> None:
         self._response = response
         self.url = response.url
         self.code = response.status_code
         self.headers = response.headers
+        self._schema = schema
         try:
             self._handle_json_response()
         except json.JSONDecodeError:
             self._handle_generic_response()
 
     def _handle_json_response(self) -> None:
         self.data = self._response.json()
+        if self._schema:
```

Comment:
is not None

Reply:
It is implied as `None` is a falsy value